v2/src/components/appInfoForm/index.tsx

```diff
@@ -33,14 +33,47 @@ type State = {
     netlifyApiRouteUsed: boolean,
     showWebsiteBasePath: boolean,
     showAPIBasePath: boolean
+    firstAppInfoForm?: boolean
     // TODO: Add more fields here
 };
 
+type ResubmitParams = {
```

Comment:
Add comments above the changes you made to explain what they do

Reply:
added

Reply:
Add more context to your comments

Reply:
its added